ci/validate-formula.sh

```diff
@@ -0,0 +1,84 @@
+#! /bin/bash
+
+# this script is intended to be executed via PRs travis CI
+set -e
+
+## this will generate the pillar
+function generate_cluster_pillar {
+  METHOD="$1"
+  cat > test/pillar/cluster.sls <<EOF
+  cluster:
+    name: 'hacluster'
+    $METHOD
+    remove: hana02
```

Comment:
I would update this to `hana03` it makes easier to understand the concept in my opinion

Reply:
ok